src/main/scala/org/allenai/pipeline/Pipeline.scala

```diff
@@ -205,11 +251,12 @@ trait Pipeline extends Logging {
     }
   }
 
-  def dryRun(outputDir: File, rawTitle: String): Iterable[Any] = {
-    val outputs = persistedSteps.toList
-    val title = s"${rawTitle.replaceAll("""\s+""", "-")}-dryRun"
+  def dryRun(outputDir: File, rawTitle: String, targets: Iterable[(String, Producer[_])] = persistedSteps.toList): Unit = {
```

Comment:
DryRun can also take a list of steps and will only output the diagram for the sub-pipeline with those steps